### ISSUE TITLE
IDE ACE CDN usage

### ISSUE THREAD

**Issue by zsimpson** [appsoma/welder#46]
When I use the browserified version of ACE it is super duper slow on Windows Chrome debugger during file load.  Mysteriously not so slow on other OS or browsers.  But since this is what I use everyday it is a big pain in the ass

I temporarily changed header,jade to use a CDN version of ace and commented out the requires in the ide.js file.

Agustin says he thinks there's a better way so I'm assigning this to him.

**Comment by kdemarest** [comment]
I feel like all this is taking way too much time.

Lets just put up some instructions to use parrot or your own ssh. Giving an
ide has always been a can of worms. We need to stop giving this so much
effort

On Monday, June 1, 2015, Zack Booth Simpson <notifications@github.com>
wrote:

> When I use the browserified version of ACE it is super duper slow on
> Windows Chrome debugger during file load. Mysteriously not so slow on other
> OS or browsers. But since this is what I use everyday it is a big pain in
> the ass
>
> I temporarily changed header,jade to use a CDN version of ace and
> commented out the requires in the ide.js file.
>
> Agustin says he thinks there's a better way so I'm assigning this to him.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/appsoma/welder/issues/46>.
>


-- 
*Ken Demarest*
*CEO, Appsoma *
ken@appsoma.com
415.342.9731
Austin, Texas ▪ United States

**Comment by zsimpson** [comment]
I strongly disagree.  An editor is an essential part of using this thing effectively. It is working well at the moment except for this weird thing of debugging on my machine which is what this bug is about.  This bug is low priority. Marking it as such.

**Comment by AgustinCB** [comment]
Ken, I wasn't thinking in spend more time on that until finished Rubicon's
work.
